docs/howto/charm-to-python.rst

```diff
@@ -0,0 +1,126 @@
+.. _howto-migrate-to-python:
+
+Migrate from the ``charm`` plugin to the ``python`` plugin
+==========================================================
+
+The ``python`` Charmcraft plugin offers a faster, stricter means of packing an operator
+charm with a virtual environment. This guide shows how to migrate from a charm using
```

Comment:
"packing with" is slightly ambiguous, so we might want to take special care with how we express this. If I'm understanding this correctly, could we say "packing a virtual environment inside an operator charm"?